### PR TITLE
Support Highlights In Education.

### PR DESCRIPTION
**Pull request by prpatel05**
This will allow people to specify highlights during their education.

### CODE REVIEW

File: resume.hbs [hunk 1]
```diff
@@ -14,9 +14,9 @@
   
   <div id="resume">
     {{> basics }}
-    {{> skills }}
     {{> work }}
     {{> projects }}
+    {{> skills }}
```
Comment (phoinixi) [inline resume.hbs]: why are you moving the skills below?

File: style.css [hunk 2]
```diff
@@ -256,13 +256,14 @@ section .location {
...
   float: right;
-  border-radius: 4px;
+  border-radius: 5px;
 }
 
 .main-summary {
   background: #f1f8ff;
   padding: 1.2em 1em;
   margin-top: 1rem;
+  float: left;
```
Comment (phoinixi) [inline style.css]: can you explain this?

File: style.css [hunk 3]
```diff
@@ -613,7 +613,7 @@ section .location {
     margin-left: 1.5em;
   }
   #resume {
-    width: 820px;
+    width: 900px;
```
Comment (phoinixi) [inline style.css]: how it that related with supporting highlights in Education?

**Comment by prpatel05** [comment]
> This PR changes way more things than Support Highlights In Education and I am not sure why.

Whoops, accidentally merged in my own personal styling choices. 

> Also in Education usually you don't have highlights but majors and that is already possible to specify and visually they appear exactly like the highlights in the other sections, so I am not sure I understand the goal of this PR

The use-cases I had in mind were certain honors/memberships, i.e. deans list, honor societies, fraternities, etc.  Let me know if you agree with this, and if so I can fix the PR.  Otherwise feel free to close.

File: resume.hbs (hunk 1 above)
Comment (prpatel05) [reply resume.hbs]: Ignore this change, it was a mistake.

File: style.css (hunk 2 above)
Comment (prpatel05) [reply style.css]: Ignore this change, it was a mistake.

File: style.css (hunk 3 above)
Comment (prpatel05) [reply style.css]: Ignore this change, it was a mistake.

**Comment by phoinixi** [comment]
have you checked this? https://jsonresume.org/schema/

**Pull request closed without merging**